File: afc_saaspro_tax/afc_rest_apis/java/src/avalara/comms/rest/sample/model/BridgeConferenceResults.java
package avalara.comms.rest.sample.model;

import com.google.gson.annotations.SerializedName;
import java.util.ArrayList;
import java.util.List;

/**
 * Structure returned when invoking the bridge conference tax calculation APIs.
 */
public class BridgeConferenceResults {

    @SerializedName("ParticipantResults")
    private List<BridgeConferenceParticipantResult> participantResults = new ArrayList<>();

    @SerializedName("Taxes")
    private List<TaxData> taxes = new ArrayList<>();

    /**
     * Sets the participant results.
     * @param participantResults
     * @return 
     */
    public BridgeConferenceResults participantResults(List<BridgeConferenceParticipantResult> participantResults) {
        this.participantResults = participantResults;
        return this;
    }

    /**
     * Adds a participant result to the bridge conference results.
     * @param participantResultsItem
     * @return 
     */
    public BridgeConferenceResults addParticipantResultsItem(BridgeConferenceParticipantResult participantResultsItem) {
        this.participantResults.add(participantResultsItem);
        return this;
    }

    /**
     * Participant results.
     * @return participantResults
     */
    public List<BridgeConferenceParticipantResult> getParticipantResults() {
        return participantResults;
    }

    /**
     * Sets the participant Taxes.
     * @param participantResults 
     */
    public void setParticipantResults(List<BridgeConferenceParticipantResult> participantResults) {
        this.participantResults = participantResults;
    }

    /**
     * Sets the summarized Taxes for Bridge Conference.
     * @param taxes
     * @return 
     */
    public BridgeConferenceResults taxes(List<TaxData> taxes) {
        this.taxes = taxes;
        return this;
    }

    /**
     * Adds a summarized tax to the bridge conference results.
     * @param taxesItem
     * @return 
     */
    public BridgeConferenceResults addTaxesItem(TaxData taxesItem) {
        this.taxes.add(taxesItem);
        return this;
    }

    /**
     * Summarized Taxes for Bridge Conference.
     * @return taxes
     */
    public List<TaxData> getTaxes() {
        return taxes;
    }

    /**
     * Sets the summarized Taxes for Bridge Conference.
     * @param taxes 
     */
    public void setTaxes(List<TaxData> taxes) {
        this.taxes = taxes;
    }
}

File: afc_saaspro_tax/afc_rest_apis/javascript/scripts/model/BridgeConferenceResults.js
/**
 * Structure returned when invoking the bridge conference tax calculation APIs.
 */

function BridgeConferenceResults() {

    this.ParticipantResults = [];
    this.Taxes = [];

    /**
     * Sets the participant results.
     * @param participantResults Array of BridgeConferenceParticipantResult
     * @return Reference to this object for method chaining. 
     */
    this.participantResults = function (participantResults) {
        this.ParticipantResults = participantResults;
        return this;
    }

    /**
     * Adds a participant result to the bridge conference results.
     * @param participantResultsItem BridgeConferenceParticipantResult object.
     * @return Reference to this object for method chaining. 
     */
    this.addParticipantResultsItem = function (participantResultsItem) {
        this.ParticipantResults.push(participantResultsItem);
        return this;
    }

    /**
     * Participant results.
     * @return Array of BridgeConferenceParticipantResult
     */
    this.getParticipantResults = function () {
        return this.ParticipantResults;
    }

    /* Reference to this object for method chaining.*
     * Sets the participant Taxes.
     * @param participantResults Array of BridgeConferenceParticipantResult 
     */
    this.setParticipantResults = function (participantResults) {
        this.ParticipantResults = participantResults;
    }

    /**
     * Sets the summarized Taxes for Bridge Conference.
     * @param taxes Array of TaxData
     * @return Reference to this object for method chaining. 
     */
    this.taxes = function (taxes) {
        this.Taxes = taxes;
        return this;
    }

    /**
     * Adds a summarized tax to the bridge conference results.
     * @param taxesItem TaxData object.
     * @return Reference to this object for method chaining. 
     */
    this.addTaxesItem = function (taxesItem) {
        this.Taxes.push(taxesItem);
        return this;
    }

    /**
     * Summarized Taxes for Bridge Conference.
     * @return Array of TaxData
     */
    this.getTaxes = function () {
        return this.Taxes;
    }

    /* Reference to this object for method chaining.*
     * Sets the summarized Taxes for Bridge Conference.
     * @param taxes Array of TaxData 
     */
    this.setTaxes = function (taxes) {
        this.Taxes = taxes;
    }
}
